lib/sentry/check_in.ex

```diff
@@ -58,7 +58,8 @@ defmodule Sentry.CheckIn do
                 optional(:max_runtime) => number(),
                 optional(:failure_issue_threshold) => number(),
                 optional(:recovery_threshold) => number(),
-                optional(:timezone) => String.t()
+                optional(:timezone) => String.t(),
+                optional(:owner) => String.t()
```

Comment:
**Blocking**: we need to mention that this is a new field available from the next version.